recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-multimedia/pulseaudio/pulseaudio_%.bbappend

```diff
@@ -0,0 +1,7 @@
+do_install_append () {
```

Comment:
is this considered as hack? should we put a comment on this why we need to behave this way?

Reply:
bump